modules/apps/27-interchain-accounts/controller/keeper/events.go

```diff
@@ -0,0 +1,27 @@
+package keeper
```

Comment:
for now I duplicated controller/keeper/events.go to avoid calling into a different keeper package and also to not break current conventions.